### ISSUE TITLE
Incorrect heading level for appointment wait time widget

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12070]
## Describe the defect
On VAMC locations pages, there is an appointment wait time widget within some of the health services accordions. This widget has a heading that is current an H3 but should be an H4 to properly fit the heading structure of the page.

## To Reproduce
1. Go to a VAMC location page, like [Captain James A. Lovell Federal VA Health Care Center](https://www.va.gov/lovell-federal-health-care-va/locations/captain-james-a-lovell-federal-health-care-center/)
2. Scroll down to Health services section
3. Expand an accordion until you find one with a heading that says, "Average number of days to get an appointment", this is the wait time widget.
4. Use the dev tools inspector, or a headings browser plugin to identify that this accordion title is an H3 as well as the "Average number of days to get an appointment" heading, which is incorrect.

## AC / Expected behavior

- [x] Update "Average number of days to get an appointment" heading to be an H4
- [ ] Accessibility review 

## Screenshots
![image](https://user-images.githubusercontent.com/106678594/209004855-2fd5514b-1b81-42b9-88a5-1ab75566009d.png)

**Comment by davidmpickett** [comment]
Removing the Lovell label as this is a problem on all VAMC pages

**Comment by CherylEvans** [comment]
This ticket was recreated in va.gov-team # [64836](https://app.zenhub.com/workspaces/collaboration-cycle-reviews-61d89e9e5f4c120011a0ad37/issues/gh/department-of-veterans-affairs/va.gov-team/64836)

**Comment by laflannery** [comment]
This is confirmed on Prod: 
![image](https://github.com/user-attachments/assets/9af13d94-2f5a-42ec-a54e-74f28d9bf171)
